seq_analysing_utilite/utilites/module2.py

```diff
@@ -0,0 +1,134 @@
+    '@SRX079805': ('TAGGGTTGTATTTGCAGATCCATGGCATGCCAAAAAGAACATCGTCCCGTCCAATATCTGCAACATACCAGTTGGTTGGTA', '@;CBA=:@;@DBDCDEEE/EEEEEEF@>FBEEB=EFA>EEBD=DAEEEEB9)99>B99BC)@,@<9CDD=C,5;B::?@;A'),
+    '@SRX079806': ('CTGCCGAGACTGTTCTCAGACATGGAAAGCTCGATTCGCATACACTCGCTGAGTAAGAGAGTCACACCAAATCACAGATT', 'E;FFFEGFGIGGFBG;C6D<@C7CDGFEFGFHDFEHHHBBHHFDFEFBAEEEEDE@A2=DA:??C3<BCA7@DCDEG*EB'),
+    '@SRX079807': ('CATTATAGTAATACGGAAGATGACTTGCTGTTATCATTACAGCTCCATCGCATGAATAATTCTCTAATATAGTTGTCAT', 'HGHHHHGFHHHHFHHEHHHHFGEHFGFGGGHHEEGHHEEHBHHFGDDECEGGGEFGF<FGGIIGEBGDFFFGFFGGFGF'),
+    '@SRX079808': ('GACGCCGTGGCTGCACTATTTGAGGCACCTGTCCTCGAAGGGAAGTTCATCTCGACGCGTGTCACTATGACATGAATG', 'GGGGGFFCFEEEFFDGFBGGGA5DG@5DDCBDDE=GFADDFF5BE49<<<BDD?CE<A<8:59;@C.C9CECBAC=DE'),
+    '@SRX079809': ('GAACCTTCTTTAATTTATCTAGAGCCCAAATTTTAGTCAATCTATCAACTAAAATACCTACTGCTACTACAAGTATT', 'DACD@BEECEDE.BEDDDDD,>:@>EEBEEHEFEHHFFHH?FGBGFBBD77B;;C?FFFFGGFED.BBABBG@DBBE'),
+    '@SRX079810': ('CCTCAGCGTGGATTGCCGCTCATGCAGGAGCAGATAATCCCTTCGCCATCCCATTAAGCGCCGTTGTCGGTATTCC', 'FF@FFCFEECEBEC@@BBBBDFBBFFDFFEFFEB8FFFFFFFFEFCEB/>BBA@AFFFEEEEECE;ACD@DBBEEE'),
+    '@SRX079811': ('AGTTATTTATGCATCATTCTCATGTATGAGCCAACAAGATAGTACAAGTTTTATTGCTATGAGTTCAGTACAACA', '<<<=;@B??@<>@><48876EADEG6B<A@*;398@.=BB<7:>.BB@.?+98204<:<>@?A=@EFEFFFEEFB'),
+    '@SRX079812': ('AGTGAGACACCCCTGAACATTCCTAGTAAGACATCTTTGAATATTACTAGTTAGCCACACTTTAAAATGACCCG', '<98;<@@@:@CD@BCCDD=DBBCEBBAAA@9???@BCDBCGF=GEGDFGDBEEEEEFFFF=EDEE=DCD@@BBC')
+    }
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+def gc_bounds_12(*args: Tuple[float, float]) -> float:
+    result = {}
+    for key, (sequence, quality) in seqs.items():
+        gc = sequence.count("G") + sequence.count("C")
+        gc_cont = gc / len(sequence) * 100
+        result[key] = gc_cont
+    return result
+
+
+def length_bounds_12(*args: Union[Tuple[int, int], List[int], int]) -> int:
+    result = {}
+    for key, (sequence, quality) in seqs.items():
+        len_seq = len(sequence)
+        result[key] = len_seq
+    return result
+
+
+def quality_threshold_12(int) -> str:
+    result = {}
+    for key, (sequence, quality) in seqs.items():
+        q_score = (sum(ord(q)-33 for q in quality)/len(quality))
+        result[key] = q_score
+=======
+phred = {
+    '!': 0,
+    '"': 1,
+    '#': 2,
+    '$': 3,
+    '%': 4,
+    '&': 5,
+    "'": 6,
+    '(': 7,
+    ')': 8,
+    '*': 9,
+    '+': 10,
+    ',': 11,
+    '-': 12,
+    '.': 13,
+    '/': 14,
+    '0': 15,
+    '1': 16,
+    '2': 17,
+    '3': 18,
+    '4': 19,
+    '5': 20,
+    '6': 21,
+    '7': 22,
+    '8': 23,
+    '9': 24,
+    ':': 25,
+    ';': 26,
+    '<': 27,
+    '=': 28,
+    '>': 29,
+    '?': 30,
+    '@': 31,
+    'A': 32,
+    'B': 33,
+    'C': 34,
+    'D': 35,
+    'E': 36,
+    'F': 37,
+    'G': 38,
+    'H': 39,
+    'I': 40
+}
+=======
```

Comment:
This seems to be excessive, as the ord function does it and is implied in the code above. I assume, that this dictionary was in the first version of the script, which was further updated with ord, but the dictionary was not properly deleted while solving the merge confilct. Could be revised for the owner's clarity.